packages/playground/src/pages/Helpers/Helpers.js

```diff
@@ -24,52 +24,44 @@ getPaymentMethods({ amount, shopperLocale }).then(async paymentMethodsResponse =
         showPayButton: true
     });
 
-    // Adyen Giving
-    window.donation = new Donation(window.checkout, {
+    // Adyen New Giving
+    window.new_donation_fixed = new Donation(window.checkout, {
         onDonate: (state, component) => {
             console.log({ state, component });
-            setTimeout(() => component.setStatus('ready'), 1000);
-        },
-        url: 'https://example.org',
-        amounts: {
-            currency: 'EUR',
-            values: [50, 199, 300]
-        },
-        disclaimerMessage: {
-            message: 'By donating you agree to the %{linkText} ',
-            linkText: 'terms and conditions',
-            link: 'https://www.adyen.com'
+            setTimeout(() => component.setStatus('success'), 1000);
         },
-        backgroundUrl:
-            'https://www.patagonia.com/static/on/demandware.static/-/Library-Sites-PatagoniaShared/default/dwb396273f/content-banners/100-planet-hero-desktop.jpg',
-        description: 'Lorem ipsum...',
-        logoUrl: 'https://i.ebayimg.com/images/g/aTwAAOSwfu9dfX4u/s-l300.jpg',
-        name: 'Test Charity',
+        donation: { type: 'fixedAmounts', currency: 'EUR', values: [50, 199, 300, 500, 1000] },
```

Comment:
The fixed amounts / round up feature - Is it only configurable on the UI side? or is it something that comes from the backend? What will come from the backend?

Reply:
Most of them will come from the backend, except for the `commercialTxAmount`, `showCancelButton` and callbacks.